app/endpoint/set-callback/SetCallbackEndpoint.ts

```diff
@@ -0,0 +1,33 @@
+import {HttpStatusCode, IHttp, IModify, IPersistence, IRead} from '@rocket.chat/apps-engine/definition/accessors';
+import {ApiEndpoint, IApiEndpointInfo, IApiRequest} from '@rocket.chat/apps-engine/definition/api';
+import {IApiResponseJSON} from '@rocket.chat/apps-engine/definition/api/IResponse';
+import AppCacheHandler from '../../local/app/cache-strategy/AppCacheHandler';
+import validateRequest from './ValidateSetCallbackEndpoint';
+
+export class SetCallbackEndpoint extends ApiEndpoint {
+    public path = 'setCallback';
+
+    public async put(
+        request: IApiRequest,
+        endpoint: IApiEndpointInfo,
+        read: IRead,
+        modify: IModify,
+        http: IHttp,
+        persis: IPersistence,
+    ): Promise<IApiResponseJSON> {
+
```

Comment:
Is the auth check in a subsequent PR?

Reply:
the /checkSecret endpoint is on the next PR: https://github.com/Ilhasoft/Rocket.Chat.App-RapidPro/pull/5